js/browzine.js

```diff
@@ -1,6 +1,6 @@
 // Define Angular module and whitelist URL of server with Node.js script
   var app = angular.module('viewCustom', ['angularLoad'])
-    .constant('nodeserver', "https://yourserver.edu")
+    .constant('nodeserver', "https://apiconnector.thirdiron.com/v1/libraries/your_browzine_library_id")
```

Comment:
I thought I could have `nodeserver` simply be `https://apiconnector.thirdiron.com/v1/libraries/` , and then have another constant - say `library_id` - be defined later, so that someone integrating this would just need to change what `library_id` is if they use the apiconnector server.

However, I don't have an instance of Primo to use, so I can't test out how that would work.  Even this approach should simplify setup instructions, though.

Thanks!